plugins/gulp/gulp.plugin.zsh

```diff
@@ -20,7 +20,7 @@
 # in the current directory.
 #
 function $$gulp_completion {
-    compls="$(grep -Eo "gulp.task\((['\"](([a-zA-Z0-9]|-)*)['\"],)" gulpfile.js 2>/dev/null | grep -Eo "['\"](([a-zA-Z0-9]|-)*)['\"]" | sed s/"['\"]"//g | sort)"
+    compls=$(gulp --tasks-simple >/dev/null | sort)
```

Comment:
This should be `2>/dev/null`, and the `| sort` can be dropped because it's already done by `compadd`.